book/src/node-driver-registrar.md

```diff
@@ -10,6 +10,7 @@
 
 Latest stable release | Branch | Min CSI Version | Max CSI Version | Container Image | [Min K8s Version](kubernetes-compatibility.md#minimum-version) | [Max K8s Version](kubernetes-compatibility.md#maximum-version) | [Recommended K8s Version](kubernetes-compatibility.md#recommended-version) |
 --|--|--|--|--|--|--|--
+[node-driver-registrar v2.4.0](https://github.com/kubernetes-csi/node-driver-registrar/releases/tag/v2.4.0) | [release-2.4](https://github.com/kubernetes-csi/node-driver-registrar/tree/release-2.4) | [v1.0.0](https://github.com/container-storage-interface/spec/releases/tag/v1.5.0) | - | k8s.gcr.io/sig-storage/csi-node-driver-registrar:v2.4.0 | v1.13 | -
```

Comment:
Also release v2.5.0 just came out yesterday

Reply:
added this too.. thanks !